// expected exit code is 5

Integer foo() {
  return 5;
}

exit(foo());